src/main/resources/assets/js/application/interviewer/tdprInterviewerModuleConfig.js

```diff
@@ -18,7 +19,7 @@ define(['angular' 
                 }
             }); 
 
-          $urlRouterProvider.otherwise("/interviewer/home/1");
+          $urlRouterProvider.otherwise("/interviewer/1/home");
```

Comment:
This should probably be removed now - or changed to point to the recruiter context.